ui/components/multichain/asset-picker-amount/asset-picker/asset-picker.tsx

```diff
@@ -107,8 +122,23 @@ export function AssetPicker({
     return undefined;
   };
 
+  const [isSelectingNetwork, setIsSelectingNetwork] = useState(false);
+
   return (
     <>
+      {networkProps && (
```

Comment:
is the networkProps not being passed down yet by any other component and we can only see how it would look in the storybook?

Reply:
That's right. I have a separate PR for passing it in from the Bridge page but I'll attach a demo here too

Reply:
https://github.com/user-attachments/assets/a956b9c8-b7b1-4b1f-b2b1-0e7a2d4454bf

Reply:
Thank you 🙏